config/config.php

```diff
@@ -7,10 +7,10 @@
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
-    $containerConfigurator->import(__DIR__ . '/../packages/**/config/config.*');
-    $containerConfigurator->import(__DIR__ . '/../rules/**/config/config.*');
+    $containerConfigurator->import(__DIR__ . '/../packages/*/config/config.*');
+    $containerConfigurator->import(__DIR__ . '/../rules/*/config/config.*');
```

Comment:
does rector still support `.yaml` based rules, or can this be restricted to `config.php` ?

Reply:
The latter, we use only `*.php` now

Reply:
Ok, adjusted the patter. i guess this will fuether speedup the bootstrapping a bit